lib/std/zig/system/NativePaths.zig

```diff
@@ -33,6 +33,10 @@ pub fn detect(arena: Allocator, native_target: std.Target) !NativePaths {
                     break;
                 };
                 try self.addFrameworkDir(framework_path);
+            } else if (mem.startsWith(u8, word, "-F")) {
+                // Ignore this argument.
+                // Nix provides the same framework path through -iframework
```

Comment:
There is no harm in including duplicated paths, and it's strictly safer than outright ignoring `-F` arg. What if suddenly the sets are disjoint between everything passed via `-iframework` and `-F`?

Reply:
Valid point. I updated the PR to include that path instead of ignore it